### ISSUE TITLE
Unable to overwrite poster if it is already in the folder

### ISSUE THREAD

**Issue by myxtrastufff** [DoctorD1501/JAVMovieScraper#85]
If I have a movie in a folder and the scraper finds the movie. It is unable to overwrite the original movie poster. Currently the files are on a CIFS share on a nas. I have tried to move the folder to local desktop of windows 7 computer and run the jar file from a command line as admin. Checking overwrite fanart and poster makes no difference. If I have everything in the folder (Movie, nfo, fanart, poster) everything is overwritten except poster and movie file.. Dates change to todays date except those to files.

Current workaround: scrape the movie click open file delete the poster.jpg the click write file data and everything is perfect.

**Comment by DoctorD1501** [comment]
I'm not able to reproduce this bug, or at least what is happening is I think expected behavior.

I had "Write fanart and poster files" checked and also "overwrite fanart and poster files" checked.

If it is not overwriting the poster file, it's because the poster file does not need to be overwritten since it is the same one as you already have on disk. I have a check to see if the poster is loaded from a poster.jpg, and if it is, when rewriting the file out, it doesn't write it out again unless you have scraped a movie with a poster.

What are the exact preferences you have checked? How about the name of the scraper used?

**Comment by myxtrastufff** [comment]
I have write fanart and poster, overwrite fanart and poster, write actor images, and write extra fan art when writing data....., and rename movie file. I already have most of my stuff scraped from data18 with the xbmc data 18 scraper. Your scraper brings in a much better poster image. higher resolution. And I use it to rename the folder to movie name with year. So if I scrape a movie with the standalone scraper it pulls in the better image but when I click write it doesn't overwrite the old image. It does everything else. Like change the folder name and overwrites the .nfo and the actor images. Other than it being the old crapy poster when I go back to the movie, I notice that the file size is still very small and the modified date is from the previous xbmc scraping.

**Comment by DoctorD1501** [comment]
Ah, when you explain your use case, the problem makes more sense. I uploaded a new build to mediafire which I think should address this issue. Can you download it and let me know if it works for you?

Thanks.

**Comment by myxtrastufff** [comment]
I apologize as I can not figure out how to find your release. I signed up for mediafire but I am unfamiliar with how the app works. I was able to follow your main release in mediafire but unable to find this specific release to test. Can you post a link for me or explain how to find this release? Thanks!

**Comment by DoctorD1501** [comment]
No signup needed. Here's the link to the newest beta build:

 http://www.mediafire.com/download/pm3d2yl49qa99fe/JAVMovieScraper.jar

**Comment by myxtrastufff** [comment]
Unfortunately it did not fix it. I still see the old small poster after a scrape and then write data. I kept the same settings and settings file. Do I need to change anything or delete the settings file?

**Comment by DoctorD1501** [comment]
I guess you can try deleting the settings file. Couldn't hurt. I've tested overwriting a poster and on my end and it's definitely possible.

What about the poster file itself? Does it have a read-only flag on it? Do you have sufficient permissions to modify the file? If it's on a NAS, sometimes the permissions can get kind of annoying to get just right.

**Comment by myxtrastufff** [comment]
I deleted the settings file and let it create a new one with default settings. Still didnt work. As for permissions I tried this. I created a new folder on my windows 7 desktop. Named it a movie name. Then I scraped it. Then I changed the name of the folder to a different movie name and then rescraped. results, Actors folder added new actors but did not delete the old actors.So now it includes movie 1 actors and movie 2 actors.  extrafanart folder did not update. the movie 1 jpg's were in it  from first scrape. nfo file was replaced with movie 2 nfo. fanart.jpg was replaced with movie 2. poster.jpg was not replaced it is still movie 1. So I cant think it would be permissions issue. Is there a way to have your program "look" for *poser.jpg and delete it before writing the new poster? Same with actors folder and extrafanart folder. Before i tried this I was thinking it was the rename part where the program renames the folder to add the year or whatever you select in the rename settings. But I did not have that checked this time. I left it all at default settings.
